bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -299,7 +320,7 @@
 	</channel-type>
 
 	<channel-type id="deviceMistLevelType">
-		<item-type>Number:Dimensionless</item-type>
+		<item-type unitHint="one">Number:Dimensionless</item-type>
```

Comment:
Don;t think you need this unit hint at all.

Reply:
For the unitHints as it displays for the item's, correctly with the configuration as is, and won't cause issues for existing users I have left as is.